impeller/renderer/backend/gles/buffer_bindings_gles.cc

```diff
@@ -312,6 +314,9 @@ bool BufferBindingsGLES::BindUniformBuffer(const ProcTableGLES& gl,
             );
             continue;
         }
+        VALIDATION_LOG << "Size " << member.size
```

Comment:
Can't this information be combined in the validation log below?

Reply:
This one is specific for floats.  The one below doesn't necessarily care about sizes, and when I hit this it was a little confusing at first about why a float wouldn't be supported.